### PR TITLE
Automate prod channel

### PR DESCRIPTION
**Pull request by mksanger**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -27,10 +28,16 @@ variables:
...
+  MASTER_BUILD_DIR: /tmp/conda_artefacts/prod
+  MASTER_COMPARE_BRANCH: old_master
+  #
   # TEST
   #
   TEST_ENV_PREFIX: $CONDA_DIR/envs/test_env
-  PROD_WSI_CONDA_CHANNEL: https://dnap.cog.sanger.ac.uk/npg/conda/prod/generic
 
 include:
   - .gitlab_include/mirror.yml
```
Comment (dkj) [inline .gitlab-ci.yml:43]:
```suggestion
  - .gitlab_include/mirror.yml
  - .gitlab_include/master_commit.yml
```
Does this need adding in this section?

File: scripts/gitlab_build.sh [hunk 2]
```diff
@@ -9,5 +9,5 @@ rsync -rlv $2 $3 # $CHANNEL_DIR $BUILD_DIR
 
 ./bin/recipebook --changes origin/$1 recipes | ./bin/build \
  --recipes-dir $CI_PROJECT_DIR --artefacts-dir $3 --conda-build-image $CONDA_IMAGE \
- --build-channel $WSI_CONDA_CHANNEL --conda-uid 1001 --conda-gid 32001 \
- --verbose --remove-container
+ --build-channel $4 --conda-uid 1001 --conda-gid 32001 \
+ --verbose --remove-container #$WSI_CONDA_CHANNEL
```
Comment (kjsanger) [inline scripts/gitlab_build.sh:13]: Maybe remove commented variable

Comment (kjsanger) [reply scripts/gitlab_build.sh:13]: Happy to merge, though

Comment (mksanger) [reply scripts/gitlab_build.sh:13]: The comments are to show what the numbered command line options are.  They are present in all of the scripts, but if there is a better (or standard) way of documenting that, I can make general change.

Comment (kjsanger) [reply scripts/gitlab_build.sh:13]: Heh. That was something I was going to open as a new (minor) issue - assigning the positional arguments to named variables, as you suggest. Meanwhile let's keep the comment.

**Pull request merged**